<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(155, 194, 231);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>251</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Filmes disponíveis</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {	
	color: rgb(255, 255, 255);
	background-color: rgb(176, 176, 176);
	border: 2px solid rgb(166, 166, 166); 
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(166, 166, 166);
	border: 2px solid rgb(138, 138, 138); 
}

QPushButton:pressed {
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(255, 165, 24); 
}</string>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {	
	color: rgb(255, 255, 255);
	background-color: rgb(176, 176, 176);
	border: 2px solid rgb(166, 166, 166); 
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(166, 166, 166);
	border: 2px solid rgb(138, 138, 138); 
}

QPushButton:pressed {
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(255, 165, 24); 
}</string>
    </property>
    <property name="text">
     <string>Próximo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {	
	color: rgb(255, 255, 255);
	background-color: rgb(176, 176, 176);
	border: 2px solid rgb(166, 166, 166); 
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(166, 166, 166);
	border: 2px solid rgb(138, 138, 138); 
}

QPushButton:pressed {
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(255, 165, 24); 
}</string>
    </property>
    <property name="text">
     <string>Informações</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>357</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
